src/flash/flashTask.ts

```diff
@@ -98,8 +98,10 @@ export class FlashTask {
       powershellPath !== ""
     ) {
       flashExecution = await this._wslFlashExecution();
-    } else {
+    } else if (isUart) {
```

Comment:
Could use a switch with case from the flashType Enum here instead of multiple if else if